### PR TITLE
Implement new Stripe Checkout flow

### LINKED ISSUE #1217: Implement new Stripe Checkout flow
Based on Backend changes from https://github.com/all3dp/printing-engine/issues/1344

- Request session id
- redirect to Stripe Checkout
- Success page must not include order specific information but should be able to always render

### LINKED ISSUE #1380: Refactor SelectField to use PositioningPortal

### PR DESCRIPTION
**Pull request by alex-all3dp**
Closes #1217 

The new [Stripe Checkout](https://stripe.com/docs/payments/checkout) supports SCA, which is required by EU law by Sep. 14th.

Breaking change which requires the backend changes from https://github.com/all3dp/printing-engine/pull/1378 

- [ ] The code does at least what is defined in the ticket and does not affect any other functionality
- [ ] The PR has a meaningful title
- [ ] There is a link to the issue-id
- [ ] There are no 'hacks' or shortcuts refactoring is preferred
- [ ] The happy case of the app was tested at least once manually

### CODE REVIEW

File: src/app/container/success-page.js [hunk 1]
```diff
@@ -32,29 +27,23 @@ import orderStarted from '../../asset/icon/order-started.svg'
...
-          />
-          <Paragraph>
+          <Paragraph classNames={['u-margin-bottom-xl']}>
             You should receive an order confirmation email from us shortly. We will also let you
             know when we have received the tracking number for your print from the manufacturer.
             Your order will be produced by:
           </Paragraph>
 
-          <ProviderTeaser classNames={['u-margin-bottom-xl']}>
+          {/* <ProviderTeaser classNames={['u-margin-bottom-xl']}>
```
Comment (henry-all3dp) [inline src/app/container/success-page.js]: I would remove that since it will not be used

File: src/app/container/cart-page.js [hunk 2]
```diff
@@ -57,6 +56,8 @@ import copyIcon from '../../asset/icon/copy.svg'
 import backIcon from '../../asset/icon/back.svg'
 import zoomInIcon from '../../asset/icon/zoom-in.svg'
 
+const stripe = global.window.Stripe(config.stripePublicKey)
```
Comment (dreampulse) [inline src/app/container/cart-page.js]: Usually we encapulate the access to global-variables to the `service` layer. I would update the existing `service/stripe` file

Comment (alex-all3dp) [reply src/app/container/cart-page.js]: done

**Comment by dreampulse** [comment]
If you merge this PR with `dev/store-core-state` it should work

**Pull request merged**